cli/cmd/honeyvent.go

```diff
@@ -45,6 +45,10 @@ const (
 	// disable telemetry sent to Honeycomb
 	DisableTelemetry = "LW_TELEMETRY_DISABLE"
 
+	// HomebrewInstall is an environment variable that denotes the
+	// install method was via homebrew package manager
+	HomebrewInstall = "LW_HOMEBREW_INSTALL"
```

Comment:
This is great!! Though, two things:
* Could we move this environment variable out of the `honeyvent.go` file and instead put it inside `cli_unix.go`
* After moving this variable, could we use it here https://github.com/lacework/go-sdk/blob/master/cli/cmd/cli_unix.go#L39